marklogic-rdf4j/build.gradle

```diff
@@ -41,9 +41,9 @@ jacocoTestReport {
 }
 
 dependencies {
-    compile('org.eclipse.rdf4j:rdf4j-runtime:2.4.0-M3')
+    compile('org.eclipse.rdf4j:rdf4j-runtime:2.4.2')
 
-    compile('com.marklogic:marklogic-client-api:4.1.0') {
+    compile('com.marklogic:marklogic-client-api:4.1.1') {
         exclude(group: 'org.slf4j')
```

Comment:
Since 4.1.2 has been released, do you want to update this ? Looks good, otherwise